tracks/r/exercises/raindrops.md

```diff
@@ -0,0 +1,60 @@
+### Possible solution
+
+This solution is very intuitive and follows step-by-step the instructions in the exercise
+
+```r
+raindrops <- function(number) {
+  rain <- ""
+
+  if (number %% 3 == 0)
+    rain <- append(rain, "Pling")
+  
+  if (number %% 5 == 0)
+    rain <- append(rain, "Plang")
+  
+  if (number %% 7 == 0)
+    rain <- append(rain, "Plong")
+  
+  if (!(number %% 3 == 0 | number %% 5 == 0 | number %% 7 == 0)) # alternatively: if(rain == "")
+    return(as.character(number))
+  
+  paste(rain, collapse = "")
+}
+
+```
+
+The learning goal here is (I guess) to avoid unnecessary conversion of inputs as strings and to account for inputs that are factors of more than one if-statement (e.g. 15).
+
+### Common suggestions
+
+- avoid a string transformation of the input at the beginning
+- go through the task iteratively to account for all possible factors
```

Comment:
What is this suggestion's intent? The tests will have already ensured this by the time they submit their solution.

Reply:
Ah sorry, the comment is a bit vague. I meant to say that if students struggle with how to best approach this problem, one could point them to the iterative solution above where one specifically states each step (if condition) that the input has to pass. This way, I think, students will just follow the instructions step-by-step and put them into the function

Reply:
If learners should see this suggestion, it's better placed in the [ReadMe](https://github.com/exercism/r/blob/master/exercises/raindrops/README.md), IMHO.

Reply:
Suggestion to resolve this: 7ddaa32.